Collections_HW/homework2/collections/DisplayStudentFullName.java

```diff
@@ -0,0 +1,53 @@
+package homework2.collections;
+
```

Comment:
Let's rename the class. DisplayStudentFullName is more like to be a method name.

Reply:
Yees, now I get it. It's better not using verbs in class names. Thank you. 
StudentService will be better?